std/assembly/math.ts

```diff
@@ -22,8 +22,21 @@ import {
 //
 // Applies to all functions marked with a comment referring here.
 
-// TODO: sin, cos, tan
+// TODO: sin, cos, tan for Math namespace
 
+var volatile32: f32;
+var volatile64: f64;
+
+@inline /** @internal */
+function FORCE_EVAL<T>(expr: T): void {
```

Comment:
If I understood this correctly, in C, this raises runtime flags if the value is inexact or similar, a mechanism that WASM doesn't have. The musl-wasm fork disabled it, so we might not need it either. Please correct me if I'm wrong :)

Reply:
Yes. This trick forcing some fenv fpu flags in portable way (don't modify it directly) like overflow/underflow and etc. You can read about it more [here](https://wiki.musl-libc.org/mathematical-library.html). In my research adding `FORCE_EVAL` macros improve speed when argument close to subnormals in next iterations. But may be this subjective?

Reply:
I see, but does it actually work for WASM?

Reply:
Not for all platforms but should

Reply:
Thinking about more I decide remove this because this cause to non-determinism between platforms